### PR TITLE
Backdrill via offset support

### PR DESCRIPTION
**Pull request by svandenb-dev**
offset with backdrill via support

### CODE REVIEW

File: pyaedt/edb_core/edb_data/padstacks_data.py [hunk 1]
```diff
@@ -1062,26 +1062,31 @@ def backdrill_top(self):
         Returns
         -------
         tuple
-            Tuple of the layer name and drill diameter.
+            Tuple of the layer name, drill diameter and offset if it exists.
```
Comment (PipKat) [inline pyaedt/edb_core/edb_data/padstacks_data.py:1065]:
```suggestion
            Tuple of the layer name, drill diameter, and offset if it exists.
```
We always use the Oxford comma, which simply means placing a comma before the "and" or "or" in a list of items.

File: pyaedt/edb_core/edb_data/padstacks_data.py [hunk 2]
```diff
@@ -1107,26 +1119,31 @@ def backdrill_bottom(self):
         Returns
         -------
         tuple
-            Tuple of the layer name and drill diameter.
+            Tuple of the layer name, drill diameter and drill offset if it exists.
```
Comment (PipKat) [inline pyaedt/edb_core/edb_data/padstacks_data.py:1122]:
```suggestion
            Tuple of the layer name, drill diameter, and drill offset if it exists.
```

File: pyaedt/edb_core/edb_data/padstacks_data.py [hunk 3]
```diff
@@ -1135,6 +1152,9 @@ def set_backdrill_bottom(self, drill_depth, drill_diameter):
             Name of the drill to layer.
         drill_diameter : float, str
             Diameter of backdrill size.
```
Comment (PipKat) [inline pyaedt/edb_core/edb_data/padstacks_data.py:1154]:
```suggestion
            Diameter of the backdrill size.
```

File: pyaedt/edb_core/edb_data/padstacks_data.py [hunk 4]
```diff
@@ -1135,6 +1152,9 @@ def set_backdrill_bottom(self, drill_depth, drill_diameter):
...
+        offset : float, str
+            Optional, if different to zero, assign an offset value for the backdrill. The stub will not stop at the
+            layer. This parameter is also called Mfg stub length in Aedt.
```
Comment (PipKat) [inline pyaedt/edb_core/edb_data/padstacks_data.py:1157]:
```suggestion
        offset : float, str, optional
            Offset for the backdrill. The default is ``0.0``. It the value is other than zero, the stub
             does not stop at the layer. In AEDT, this parameter is called "Mfg stub length".
```

Comment (MaxJPRey) [inline pyaedt/edb_core/edb_data/padstacks_data.py:1157]: @svandenb-dev Is it `Nfg` or `Mfg`?
Above you mentioned `Nfg` but it migth be `Mfg`
Just to be sure after you will have applied all the suggestion.

**Comment by ring630** [comment]
LGTM

File: pyaedt/edb_core/edb_data/padstacks_data.py (hunk 4 above)
Comment (svandenb-dev) [reply pyaedt/edb_core/edb_data/padstacks_data.py:1157]: @MaxJPRey thanks it's Mfg. fixed

**Review by maxcapodi78 — APPROVED** [review]

**Pull request merged**